package.json

```diff
@@ -154,4 +154,4 @@
     "storybook": "storybook dev -c client/storybook -p 6006 --no-open --no-version-updates",
     "build-storybook": "storybook build -c client/storybook"
   }
-}
+}
```

Comment:
It looks like your editing environment may be set up with a different end of file line break.  

Maybe check - here's some useful docs. 
https://docs.wagtail.org/en/stable/contributing/developing.html#development-on-windows